src/indra_cogex/apps/constants.py

```diff
@@ -108,3 +109,8 @@
         "and CLARE_PUSHER_CLUSTER."
     )
     pusher_app = None
+
+PYOBO_RESOURCE_FILE_VERSIONS = {
+    "mgi": "6.23",
+    "rgd": "2024-05-31",
+}
```

Comment:
Depending on exact usage, these could be replaced by what is already available via `indra.databases`

Reply:
The usage is to [look up identifiers from names/symbols](https://github.com/gyorilab/indra_cogex/blob/pubmed-rels-debug/src/indra_cogex/client/enrichment/continuous.py#L183), but not translate them to human equivalents. So I think we'll keep the implementation with pinning the versions and just make sure the resource files exist as part of the docker build.